python/leap/leap.py

```diff
@@ -0,0 +1,65 @@
+## exercism assignment:     leap.py
+## description:             evaluate whether year(s) passed in are leap years
+## use:                     leap.py year [year..]
+##
+## written by (GitHub):     cmdellinger
+## written on:              10.26.2016
+## last revised:            11.02.2016
+
+## ------
+## module import calls
+## ------
+
+import sys
+
+## ------
+## function definitions
+## ------
+
+## ------
+## function:    is_leap_year
+## inputs:      integer
+## outputs:     Boolean
+## purpose:     takes in the year as an integer and evaluates whether the year is a leap year to a Boolean value
+## ------
+
+def is_leap_year(year):
+    if year%4 == 0:
+        if year%100 == 0 and year%400 != 0:
+            return False
+        return True
+    return False
+
+## ------
+## main
+## ------
+
+## test whether an argument is passed in the command line
+if len(sys.argv) > 1:
```

Comment:
its been a while since i've done python... but [docopt](https://github.com/docopt/docopt) is a cool library that i've used in the past for command line stuff. Refactor and try your hand at using a third party library.